src/main/webapp/i18n/de/modelingExercise.json

```diff
@@ -17,12 +17,12 @@
             },
             "diagramType": "Diagrammtyp",
             "problemStatement": "<strong>Aufgabenstellung:</strong> {{ problemStatement }}",
-            "exampleSolution": "Beispiellösung",
-            "exampleSolutionExplanation": "Erklärung Beispiellösung",
-            "createExampleSolution": "Beispiellösung erstellen",
-            "exampleSolutionRequiresExercise": "Um eine Beispiellösung zu erstellen, müssen Sie zunächst die Erstellung der Textaufgabe abschließen.",
-            "exampleSolutionForModelingExercise": "Beispiellösung für Modellierungsaufgabe {{ exerciseTitle }}",
-            "saveExampleSolution": "Beispiellösung speichern",
+            "exampleSolution": "Beispiel Lösung",
```

Comment:
`Lösungsbeispiel` or `Beispiellösung` would be a better wording

Reply:
I am fine with either, however we have to be consistent in the whole user interface.